content/boards/arduino-mega-2560/index.md

```
@@ -16,6 +16,20 @@
 - 54 digital IO pins, 15 with PWM support.
 - 16 analog inputs (can be used as digital inputs).
 
+ Device | Limit | Notes |
+| ------------- | :-----------: | ----------- |
```

Markdownlint (0.37.0)

Expected: trailing_only; Actual: leading_and_trailing; Unexpected leading pipe
Table pipe style

(MD055, table-pipe-style)